src/test.ts

```diff
@@ -1,14 +1,15 @@
 import * as fs from "fs";
 import * as path from "path";
 import child_process from "child_process";
+import printDiff = require("print-diff");
 
 const __SOURCE_DIRECTORY__ = __dirname;
 const baselineFolder = path.join(__SOURCE_DIRECTORY__, "../", "baselines");
 const outputFolder = path.join(__SOURCE_DIRECTORY__, "../", "generated");
 const tscPath = path.join(__SOURCE_DIRECTORY__, "../", "node_modules", "typescript", "lib", "tsc.js");
 
 function normalizeLineEndings(text: string): string {
-    return text.replace(/\r\n?/g, "\n");
+    return text.replace(/\r\n?/g, "\n").replace(/\n*$/, "\n");
```

Comment:
The files in git doesn’t have a trailing newline at all, but my editor (VS Code) adds one when I open the file. This I always got a diff.

This can of course be turned off in my editor, but I thought it was common enough so that it would help to ignore it. I think Atom is also adding trailing newline by default.

Reply:
Yeah, sorry, saved is correct. The workflow I've been using is to first modify the baseline files to be what I want, and then work with the json files to get that result. That's when I ran into the problem, since I had saved a new baseline file.

Reply:
Um... You probably will want to try npm run build && npm run baseline-accept 😁

Reply:
Hehe, yeah, although I wasn't familiar with how the JSON files worked. So it was actually a quite nice workflow for me, since I could set up my target and then work towards that :)

Reply:
Personally still don't like this, how about adding a trailing newline through emitter.ts rather than ignoring it?